docs/tokenapi.md

```diff
@@ -167,39 +161,20 @@ With Token Vault, you can:
 * **List all tokens:** Get a complete overview of all your tokens or those issued on the network.
 * **Dive deeper:** Retrieve specific details about tokens and their outputs. (For certain systems) You can even find out who deleted a token, if applicable.
 
-**Easy access for developers:** Developers can effortlessly access the vault of a specific Token Management System (TMS) using a simple line of Go code:
-
-```go
-vault := tms.Vault()
-```
-
-This vault instance is the same one provided by the dedicated [`Vault Service`](../services/vault.md).
+The `Token Vault` is built on top of the `Transactions DB` `Audit Transactions DB` and `Tokens DB`.
 
 ## Token Selector Manager
 
-The token SDK empowers developers to select specific tokens from the vault for transactions using the `token.Selector` function. 
-This selector acts like a refined filter, allowing you to specify conditions like token type, amount, and even ownership. 
-To safeguard against double-spending, any chosen token is automatically locked until the transaction's fate is sealed – be it commitment, rejection, timeout, or manual unlock. 
+The Token SDK empowers developers to select specific tokens from the vault for transactions using the `token.Selector` function.
+This selector acts like a refined filter, allowing you to specify conditions like token type, amount, and even ownership.
+To safeguard against double-spending, any chosen token is automatically locked until the transaction's fate is sealed – be it commitment, rejection, timeout, or manual unlock.
 This ensures developers leverage the appropriate tokens while eliminating double-spending woes.
 
-Let's delve deeper. 
-To get a selector for your specific needs, simply call the `SelectorManager()` method on your TMS instance. 
-This manager provides access to new selectors using the `NewSelector(tr.Anchor)` function.
-Remember, the chosen tokens will be locked under the provided ID.
-
-Behind the scenes, these selectors are meticulously crafted by the dedicated [`Token Selector Service`](../services/selector.md).
-
 ## Signature Service
 
-The `token.SignatureService` acts as your gateway to secure transactions. 
+The `token.SignatureService` acts as your gateway to secure transactions.
 It provides access to both signature verifiers and signers, all seamlessly linked to identities retrieved from the Wallet Manager.
```

Comment:
Should there be more information here about how signers/verifiers are linked to wallets.